### PR TITLE
Fixed worldspace export flag behaviour to only bake xform at the top level prim

### PR DESCRIPTION
**Pull request by csyshing**
This PR fixed tow problems:
 - The `worldspace` flag for exporter to only bake xform at the top level prim, children prims xforms would be remained unchanged
 - The logic to handle selection objects has also been modified to match `mayaUsdPlugin` that when selecting parent node and one of the children, unselected siblings would be excluded, for example:
 ```
- parent            * selected
    - child_A       * selected
    - child_B
```
`child _B` object would be excluded when exporting.

### CODE REVIEW

**Comment by csyshing** [comment]
Jenkins test failed due to this one:
```
    ======================================================================
    FAIL: test_anonymousLayerSerialisation (testLayerManager.TestLayerManagerSerialisation)
    Tests multiple anonymous sublayers can be saved and restored.
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/S/jenkins/workspace/ecg-mayausd-branch-preflight-2020-pxrusd-mid3-interactive-linux/ecg-maya-usd/maya-usd/plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py", line 262, in test_anonymousLayerSerialisation
        self.assertIsNotNone(sublayer02)
    AssertionError: unexpectedly None
    
    ----------------------------------------------------------------------
```
Strangely this is the same reason for other PRs e.g. https://github.com/Autodesk/maya-usd/pull/1924 .

The unit test above should have been fixed (and have been merged) in https://github.com/Autodesk/maya-usd/pull/1872/commits/c65e1d71528dbb7cb10653160d0b75e6ee48e202 

Any idea?

**Comment by seando-adsk** [comment]
@csyshing Is there someone who can code review these changes? Once completed I will merge this PR.

**Comment by csyshing** [comment]
@seando-adsk , this PR has been reviewed internally in Animal Logic; on external, usually @kxl-adsk would do the review?

**Comment by seando-adsk** [comment]
@csyshing Okay. Krystian is off this week. Are you okay waiting until next week? If not we can find someone else to review.

**Comment by csyshing** [comment]
No problem @seando-adsk , it's not urgent to merge this PR.

**Review by kxl-adsk — DISMISSED** [review]

**Review by kxl-adsk — DISMISSED** [review]

**Review by kxl-adsk — APPROVED** [review]

**Pull request merged**
